websites/K/Kahoot/presence.ts

```diff
@@ -1,18 +1,16 @@
-async function findRanking(rankingSelector: HTMLElement) {
-	if (
+const presence = new Presence({
+	clientId: "958898877598146571"
```

Comment:
please explain why a new clientid was needed.

Reply:
To update the large image

Reply:
You can contact @StrikerFRFX about that

Reply:
Got it.

Reply:
What is the best way to contact them?

Reply:
Idk probably discord

Reply:
Has the assets been added now?

Reply:
I have tried contacting them, but they haven't responded yet.

Reply:
The assets have been added.
<img width="220" alt="2022-04-01 07_24_45" src="https://user-images.githubusercontent.com/32113157/161283147-b42b8073-de05-4017-b963-5ca06b243929.png">